nolabs/controllers/biobuddy/biobuddy.py

```diff
@@ -28,9 +31,25 @@ async def load_conversation(experiment_id: str,
     return feature.handle(LoadConversationRequest(experiment_id))
 
 
-@router.post('/send-message')
-async def send_message(experiment_id: str,
+@router.post('/save-message')
```

Comment:
```suggestion
@router.post('/save-message')
```
@router.post('/message')